classes/report/fields/runner/tests/coverage/cli.php

```diff
@@ -71,43 +73,46 @@ public function __toString()
 			}
 
 
-			foreach ($this->coverage->getMethods() as $class => $methods)
+			if ($this->hideClassesCoverageDetails === false)
 			{
-				$classCoverage = $this->coverage->getValueForClass($class);
-				$classPathsCoverage = $this->coverage->getPathsCoverageValueForClass($class);
-				$classBranchesCoverage = $this->coverage->getBranchesCoverageValueForClass($class);
-
-				if ($classCoverage < 1.0)
+				foreach ($this->coverage->getMethods() as $class => $methods)
 				{
-					$string .= $this->classPrompt .
-						sprintf(
-							$this->locale->_('%s: %s%s%s'),
-							$this->titleColorizer->colorize(sprintf($this->locale->_('Class %s'), $class)),
-							$this->coverageColorizer->colorize(sprintf('%s%3.2f%%', ($classPathsCoverage !== null || $classBranchesCoverage !== null ? 'Line: ' : ''), $classCoverage * 100.0)),
-							$classPathsCoverage !== null ? $this->coverageColorizer->colorize(sprintf(' Path: %3.2f%%', $classPathsCoverage * 100.0)) : '',
-							$classBranchesCoverage !== null ? $this->coverageColorizer->colorize(sprintf(' Branch: %3.2f%%', $classBranchesCoverage * 100.0)) : ''
-						) .
-						PHP_EOL
-					;
-
-					foreach (array_keys($methods) as $method)
-					{
-						$methodCoverage = $this->coverage->getValueForMethod($class, $method);
-						$methodPathsCoverage = $this->coverage->getPathsCoverageValueForMethod($class, $method);
-						$methodBranchesCoverage = $this->coverage->getBranchesCoverageValueForMethod($class, $method);
+					$classCoverage = $this->coverage->getValueForClass($class);
+					$classPathsCoverage = $this->coverage->getPathsCoverageValueForClass($class);
+					$classBranchesCoverage = $this->coverage->getBranchesCoverageValueForClass($class);
 
-						if ($methodCoverage < 1.0)
+					if ($classCoverage < 1.0)
+					{
+						$string .= $this->classPrompt .
+							sprintf(
+								$this->locale->_('%s: %s%s%s'),
```

Comment:
In order to do multinlingual transalion easily, i think that a better strategy here is to do:
```
$this->locale->_('%s: %s %s %s'),
$this->titleColorizer->colorize(sprintf($this->locale->_('Class %s'), $class)),
$this->coverageColorizer->colorize(sprintf('%s%3.2f%%', ($classPathsCoverage !== null || $classBranchesCoverage !== null ? 'Line: ' : ''), $classCoverage * 100.0)),
$classPathsCoverage !== null ? $this->coverageColorizer->colorize(sprintf('Path: %3.2f%%', $classPathsCoverage * 100.0)) : '',
$classBranchesCoverage !== null ? $this->coverageColorizer->colorize(sprintf('Branch: %3.2f%%', $classBranchesCoverage * 100.0)) : ''
```